server/index.js

```diff
@@ -16,7 +16,7 @@ app.get('/api/items', (req, res) => {
     .then((results) => {
       res.end(JSON.stringify(results));
     })
-    .catch(err => console.log('promise rejection error ', err));
+    .catch(err => res.end(err));
```

Comment:
Interesting! Why `res.end()` and not `res.send()`?

Reply:
I figured res.end works since I don't need to specify content-type. But I haven't put much thought into my server so far.